aequilibrae/paths/traffic_assignment.py

```diff
@@ -397,8 +402,7 @@ def __validate_parameters(self, kwargs) -> bool:
             raise ValueError("First you need to set the Volume-Delay Function to use")
 
         par = list(kwargs.keys())
-        if self.vdf.function == "BPR":
```

Comment:
this seems like a non-project related change...

Reply:
It is. It was a bug, which I am not sure how had not been reported yet